ibm/mas_devops/playbooks/cp4d/install-cp4d.yml

```diff
@@ -1,5 +1,38 @@
 ---
 - hosts: localhost
   any_errors_fatal: true
+  vars:
```

Comment:
Why are these being added back in?  We moved the properties into the roles a while back.